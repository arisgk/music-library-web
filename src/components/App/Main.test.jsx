/* global describe, beforeAll, it, expect */
import React from 'react';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';
import { createShallow } from '@material-ui/core/test-utils';
import Wrapper, { Main } from './Main';

Enzyme.configure({ adapter: new Adapter() });

describe('Home', () => {
  let shallow;

  beforeAll(() => {
    shallow = createShallow({ untilSelector: Main });
  });

  it('renders correctly', () => {
    const wrapper = shallow(<Wrapper />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
