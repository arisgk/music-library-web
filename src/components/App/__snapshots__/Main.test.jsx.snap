// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<div
  className="Main-container-1"
>
  <div
    className="Main-logoContainer-2"
  >
    <pure(MusicNoteIcon)
      color="primary"
      fontSize="large"
    />
  </div>
  <div
    className="Main-homeContainer-3"
  >
    <Connect(WithStyles(Home)) />
  </div>
  <div
    className="Main-controlsContainer-4"
  >
    <Connect(WithStyles(Controls)) />
  </div>
  <Connect(WithStyles(Notifications)) />
</div>
`;
